pkg/aws/cloud.go

```diff
@@ -114,9 +108,19 @@ func NewCloud(cfg CloudConfig, metricsRegisterer prometheus.Registerer) (Cloud,
 		metricsCollector.InjectHandlers(&sess.Handlers)
 	}
 
+	ec2Service := services.NewEC2(sess)
+
+	if len(cfg.VpcID) == 0 {
```

Comment:
nit: i think we should use same check here, either `len(cfg.VpcID) ==0` or `vpcID == ""`

Reply:
You were referring to this one and the one two lines below? The latter was replaced by an err check now that the infer function returns errors.